tests/test_pyrflx.py

```diff
@@ -916,30 +936,41 @@ def test_ethernet_generating_ethernet_2_vlan(self) -> None:
             b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
             b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x0a"
         )
-        self.frame.set("Destination", int("FFFFFFFFFFFF", 16))
-        self.frame.set("Source", int("0", 16))
-        self.frame.set("Type_Length_TPID", int("8100", 16))
-        self.frame.set("TPID", int("8100", 16))
-        self.frame.set("TCI", 1)
-        self.frame.set("Type_Length", int("0800", 16))
-        self.frame.set("Payload", payload)
-        self.assertTrue(self.frame.valid_message)
+        self.frame_without_refinement.set("Destination", int("FFFFFFFFFFFF", 16))
+        self.frame_without_refinement.set("Source", int("0", 16))
+        self.frame_without_refinement.set("Type_Length_TPID", int("8100", 16))
+        self.frame_without_refinement.set("TPID", int("8100", 16))
+        self.frame_without_refinement.set("TCI", 1)
+        self.frame_without_refinement.set("Type_Length", int("0800", 16))
+        self.frame_without_refinement.set("Payload", payload)
+        self.assertTrue(self.frame_without_refinement.valid_message)
         with open("tests/ethernet_vlan_tag.raw", "rb") as file:
             msg_as_bytes: bytes = file.read()
-        self.assertEqual(self.frame.bytestring, msg_as_bytes)
-
-    # ISSUE: Componolit/RecordFlux#199
+        self.assertEqual(self.frame_without_refinement.bytestring, msg_as_bytes)
 
-    # def test_ethernet_generating_ethernet_2_vlan_dynamic(self) -> None:
-    #    raise NotImplementedError
+    # test is not relevant for the python implementation
+    def test_ethernet_generating_ethernet_2_vlan_dynamic(self) -> None:
+        pass
```

Comment:
Does it make sense then to keep an empty test method? I think just the name of the test and that it is not relevant might be enough. Maybe with an explanation why it is not relevant.

Reply:
As we thought about creating a script which checks if the Python and SPARK tests are in sync, I think it will be easier to have consistently a test method on each side.

Reply:
This is a good point. In this case we could keep the emtpy test.